docker-compose.yml

```diff
@@ -130,26 +66,22 @@ services:
     image: "postgres"
     restart: always
     environment:
-      DEPLOYMENT_NAME: "DOCKER_TEST"
       POSTGRES_DB: "sempo_docker_eth"
       POSTGRES_USER: "postgres"
       POSTGRES_PASSWORD: "password"
 
   redis:
     image: "redis:alpine"
-    environment:
-      DEPLOYMENT_NAME: "DOCKER_TEST"
 
   ganache:
-    image: "trufflesuite/ganache-cli:v6.7.0"
-    command: "-l 80000000 -i 42 ${GANACHE_ACCOUNT_COMMAND}"
```

Comment:
The use of `GANACHE_SEED_DETERMINISTIC` is much nicer, but without this when you launch a dev instance in docker, the wallet generated by `generate_secrets.py` won't have access to any funds, as this is where it's fed in.

Three solutions i see:
1) Feed the private key to GANACHE_ACCOUNT_COMMAND as done previously.
Downside is that it's another step devs setting up need to know about
2) Set the dev wallet pk deterministically.
pros: less dev setup
cons: is slightly weird that there's a single dev secret that is deterministic when all the rest are random
2b)Even more so, could just have deterministic secrets/the same for the entire secrets.ini files for all dev/tests. Gets us even closer to one-click dev setup
Pros:
- fully deterministic tests
- even less dev setup
Cons:
- Could be a security risk if someone tries to use those in prod (seems unlikely)
- Could be a security risk if someone tries accidentally pastes in actual secrets into a file that is no-longer git ignored (slightly more likely, but still unlikely) 
- We still need to set custom db credentials, so some amount of manual config wrangling is always required, though config.py could default to pulling from env variables.
3) Just include a boot script in app/or eth_worker that transfers funds from one of the randomly generated wallets to the master wallet when in dev or test mode
pros: we do this in tests already
cons: another conditional logic piece that needs to fire.